### PR TITLE
[zh-cn] sync task files of  task-2

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Part of: https://github.com/kubernetes/website/issues/39983

### CODE REVIEW

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/declarative-config.md [hunk 1]
```diff
@@ -763,8 +763,8 @@ kind: Deployment
 metadata:
   annotations:
     # ...
-    # 注解中包含更新后的 image，nginx 1.16.1,
-    # 但不包含更新后的 replicas
+    # 注解中包含更新后的镜像 nginx 1.16.1,
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/declarative-config.md:766]: for convenience: 
```yaml
    # The annotation contains the updated image to nginx 1.16.1,
    # but does not contain the updated replicas to 2
```

**Review by howieyuen — COMMENTED** [review]
/approve

**Comment by windsonsea** [comment]
Thanks. The CI is failed occasionally. Need to retrigger it again.

/lgtm

**Comment by mengjiao-liu** [comment]
test timeout: forces push again to trigger the CI test.

**Comment by mengjiao-liu** [comment]
Request LGTM label again. @windsonsea @tengqm

**Pull request merged**